src/components/Subthoughts.js

```diff
@@ -227,6 +231,7 @@ export const Subthoughts = connect(({ cursorBeforeEdit, cursor, contextViews, th
             : getThoughtsRanked(contextBinding || thoughtsRanked)
```

Comment:
If `sortPreference === 'Alphabetical`, you'll want to call `getThoughts` instead of `getThoughtsRanked` to avoid an unnecessary sort by rank.